app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,266 @@
+            android:background="@color/colorPrimary" />
+
+        <RelativeLayout
+            android:id="@+id/other_categ_based_on_search"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/height_260"
+            android:layout_below="@id/line"
+            android:layout_marginTop="@dimen/margin_15"
+            android:layout_marginLeft="@dimen/margin_25"
+            android:layout_marginRight="@dimen/margin_25"
+            android:background="@drawable/background_with_radius">
+
+
+            <TextView
+                android:id="@+id/other_categ_name"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/margin_15"
+                android:layout_marginTop="@dimen/margin_15"
+                android:text="@string/any_other_category"
+                android:textColor="@color/color_e6e4e4"
+                android:textSize="@dimen/textSize_14"
+                android:textStyle="bold" />
+
+            <TextView
+
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/other_categ_name"
+                android:layout_marginLeft="@dimen/margin_13"
+                android:layout_marginTop="@dimen/margin_5"
+                android:text="@string/based_on_your_recent_searches"
+                android:textColor="@color/color_e6e4e4" />
+
+            <TextView
+                android:id="@+id/tv_seemore2"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/other_categ_name"
+                android:layout_alignParentRight="true"
+                android:layout_marginTop="@dimen/margin_190"
+                android:layout_marginRight="@dimen/margin_15"
+                android:clickable="true"
+                android:onClick="seemoreClick"
+                android:text="@string/see_more"
+                android:textColor="@color/color_e6e4e4" />
+
+            <View
+                android:layout_width="@dimen/width_60"
+                android:layout_height="@dimen/height_1"
+                android:layout_below="@id/tv_seemore2"
+                android:layout_alignParentRight="true"
+                android:layout_marginRight="@dimen/margin_14"
+                android:background="@color/color_e6e4e4" />
+
+        </RelativeLayout>
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/rv_other_categ"
+            android:layout_below="@id/similarItem_recyclerView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_100"
+            android:paddingTop="@dimen/padding_10"
+            android:paddingLeft="@dimen/padding_10"
```

Comment:
this padding should not be placed
item has to scroll horizontally by the width of the screen